cg/services/deliver_files/delivery_file_fetcher_service/delivery_file_fetcher_service.py

```diff
@@ -28,3 +32,13 @@ def __init__(
     def get_files_to_deliver(self, case_id: str) -> DeliveryFiles:
         """Get the files to deliver."""
         pass
+
+    @staticmethod
+    def _validate_delivery_has_content(delivery_files: DeliveryFiles) -> DeliveryFiles:
```

Comment:
Slight preference for having this method not return anything and just raising an error if all files are missing, but feel free to go either way

Reply:
I understand, I will keep it this way then. The other one would require an extra line of code :P